process_pr.py

```diff
@@ -2189,6 +2189,7 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             for signature in new_assign_cats
             if signature in l2_categories
         ]
+        new_l2s = list(OrderedDict.fromkeys(new_l2s))
```

Comment:
thanks @kpedro88 . I guess just using `sorted(set(new_l2s))` should work too ... right ? I guess we do not care about the order too much here so having a sorted list will be good too

Reply:
Sure, I can change to that if the order doesn't matter.

Reply:
+1
yes order here does not matter.